test/sanitizer_suppressions/ubsan

```diff
@@ -56,6 +58,7 @@ implicit-integer-sign-change:script/interpreter.cpp
 implicit-integer-sign-change:serialize.h
 implicit-integer-sign-change:test/arith_uint256_tests.cpp
 implicit-integer-sign-change:test/coins_tests.cpp
+implicit-integer-sign-change:test/fuzz/FuzzedDataProvider.h
```

Comment:
what are the steps to reproduce this?

Reply:
I don't think is needed now that we've updated `FuzzedDataProvider.h` to the most recent version. Suppression removed! :)

Reply:
`test_runner.py` on macOS, clang-11, `--with-sanitizers=fuzzer,undefined,integer`:
```
test/fuzz/FuzzedDataProvider.h:211:49: runtime error: implicit conversion from type 'int' of value -2147483648 (32-bit, signed) to type 'unsigned long long' changed the value to 18446744071562067968 (64-bit, unsigned)
SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior test/fuzz/FuzzedDataProvider.h:211:49 in 
test/fuzz/FuzzedDataProvider.h:211:47: runtime error: unsigned integer overflow: 2147483647 - 18446744071562067968 cannot be represented in type 'unsigned long long'
SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior test/fuzz/FuzzedDataProvider.h:211:47 in 
```